react/index.d.ts

```diff
@@ -2140,60 +2140,237 @@ declare namespace React {
         unselectable?: boolean;
     }
 
+    // this list is "complete" in that it contains every SVG attribute 
+    // that React supports, but the types can be improved.
+    // Full list here: https://facebook.github.io/react/docs/dom-elements.html
```

Comment:
As in, an attribute like `alignmentBaseline` may only take on certain values (including `inherit`, `auto`, `central`, and a few more). Many other attributes can be similarly typed as unions of string literals, but there is a lot of (often missing) documentation to dig through